app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -122,33 +122,28 @@ def get_referenceBranch(self, obj):
             return obj.reference_branch.slug
 
     def get_targeting(self, obj):
-        version_expr = ""
-        if obj.firefox_min_version:
-            version_expr = "{version_check} && ".format(
-                version_check=NimbusExperiment.TARGETING_VERSION.format(
-                    version=obj.firefox_min_version
-                )
-            )
+        exprs = []
 
-        channel_expr = ""
-        if obj.channel:
-            channel_expr = "{channel_check} && ".format(
-                channel_check=NimbusExperiment.TARGETING_CHANNEL.format(
-                    channel=obj.channel
+        if obj.is_desktop_experiment:
```

Comment:
There's a redundant `if obj.is_desktop_experiment` here. But, I didn't want to touch all the tests to reorder the expressions. That way, the original tests were mostly intact & untouched & still passing after this refactor. Also, it looks like the second `if obj.is_desktop_experiment` is part of a TODO to remove, so 🤷‍♂️